pyvista/core/utilities/arrays.py

```diff
@@ -259,7 +260,7 @@ def convert_array(arr, name=None, deep: bool = False, array_type=None):
     return _vtk.vtk_to_numpy(arr)
 
 
-def get_array(mesh, name, preference='cell', err: bool = False) -> pyvista.ndarray | None:
+def get_array(mesh, name, preference='cell', err: bool = False) -> pyvista_ndarray | None:
```

Comment:
The new type errors are caused by this change and are ignored here. The typing for `arrays.py` is fixed in #6856.